common/service/dynamicconfig/constants.go

```diff
@@ -24,38 +24,106 @@ package dynamicconfig
+	TimerTaskWorkerCount:                               "history.timerTaskWorkerCount",
+	TimerTaskMaxRetryCount:                             "history.timerTaskMaxRetryCount",
+	TimerProcessorGetFailureRetryCount:                 "history.timerProcessorGetFailureRetryCount",
+	TimerProcessorCompleteTimerFailureRetryCount:       "history.timerProcessorCompleteTimerFailureRetryCount",
+	TimerProcessorUpdateShardTaskCount:                 "history.timerProcessorUpdateShardTaskCount",
+	TimerProcessorUpdateAckInterval:                    "history.timerProcessorUpdateAckInterval",
+	TimerProcessorCompleteTimerInterval:                "history.timerProcessorCompleteTimerInterval",
+	TimerProcessorMaxPollInterval:                      "history.timerProcessorMaxPollInterval",
+	TimerProcessorStandbyTaskDelay:                     "history.timerProcessorStandbyTaskDelay",
+	TransferTaskBatchSize:                              "history.transferTaskBatchSize",
+	TransferProcessorMaxPollRPS:                        "history.transferProcessorMaxPollRPS",
+	TransferTaskWorkerCount:                            "history.transferTaskWorkerCount",
+	TransferTaskMaxRetryCount:                          "history.transferTaskMaxRetryCount",
+	TransferProcessorCompleteTransferFailureRetryCount: "history.transferProcessorCompleteTransferFailureRetryCount",
+	TransferProcessorUpdateShardTaskCount:              "history.transferProcessorUpdateShardTaskCount",
+	TransferProcessorMaxPollInterval:                   "history.transferProcessorMaxPollInterval",
+	TransferProcessorUpdateAckInterval:                 "history.transferProcessorUpdateAckInterval",
+	TransferProcessorCompleteTransferInterval:          "history.transferProcessorCompleteTransferInterval",
+	TransferProcessorStandbyTaskDelay:                  "history.transferProcessorStandbyTaskDelay",
+	ReplicatorTaskBatchSize:                            "history.replicatorTaskBatchSize",
+	ReplicatorTaskWorkerCount:                          "history.replicatorTaskWorkerCount",
+	ReplicatorTaskMaxRetryCount:                        "history.replicatorTaskMaxRetryCount",
+	ReplicatorProcessorMaxPollRPS:                      "history.replicatorProcessorMaxPollRPS",
+	ReplicatorProcessorUpdateShardTaskCount:            "history.replicatorProcessorUpdateShardTaskCount",
+	ReplicatorProcessorMaxPollInterval:                 "history.replicatorProcessorMaxPollInterval",
+	ReplicatorProcessorUpdateAckInterval:               "history.replicatorProcessorUpdateAckInterval",
+	ExecutionMgrNumConns:                               "history.executionMgrNumConns",
+	HistoryMgrNumConns:                                 "history.historyMgrNumConns",
+	MaximumBufferedEventsBatch:                         "history.maximumBufferedEventsBatch",
+	ShardUpdateMinInterval:                             "history.shardUpdateMinInterval",
 }
 
 const (
 	// The order of constants is important. It should match the order in the keys array above.
```

Comment:
Probably this comment is not relevant anymore.  Delete this.

Reply:
Good catch done